superset-frontend/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -122,138 +127,110 @@ const Styles = styled.div`
   }
 `;
 
-class ExploreViewContainer extends React.Component {
```

Comment:
Converted this to a function component out of frustration with changing it. Didn't have the stamina to go all the way and convert it to typescript.